posydon/binary_evol/MESA/step_mesa.py

```diff
@@ -1470,6 +1474,11 @@ def __call__(self, binary):
                                   'while the period is inside the grid.')
                 
         else:
+            if len(self.binary.state_history) > 2:
+                if self.binary.state_history[-2] == 'detached':
+                    set_binary_to_failed(self.binary)
+                    raise ValueError('CO_HMS_RLO binary outside grid and coming from detached')
+
```

Comment:
These selections have been removed with a later PR on the development branch.
They should be removed for the CO_HMS_RLO and CO_HeMS_RLO grids.

CO-HeMS does not have the extra code.

```
 if len(self.binary.state_history) > 2:
                if self.binary.state_history[-2] == 'detached':
                    set_binary_to_failed(self.binary)
                    raise ValueError('CO_HMS_RLO binary outside grid and coming from detached')
```

Reply:
removed.